### ISSUE TITLE
Remove MO LegacyEdition and replace it with MO CommunityEdition.

### ISSUE THREAD

**Issue by Layarion** [DarkPacks/SkyFactory-4#2083]
## Context

**TLDR,** I'm asking for the CommunityEdition of MatterOverdrive because the LegacyEdition is clearly unfinished and has some real buzzkills in it. The only downside this change will bring, is that the infinite power glitch will be gone.

Please replace the LegacyEdition of MatterOverdrive with the CommunityEdition, because it didn't take long in-game to figure out that LE is unfinished.

LE has many bugs, some of them that really kill the mood. Example, when you become an android, potions don't work on you anymore. So you can't use invisibility, not even with a tinkers cloak. LE let's you buy an invis cloak, but it's unusable. Completely turns your screen white because of some issue with how LE renders something. So you can't go invis, and you can't use it's own invis. You have to give up on invisibility if you want to be an android.

The CommunityEdition however, fixes this issue. It also still has active dev authors around to fix bugs, and to top it off some other issues are fixed as well. Such as LE's infinite power exploit, and CE doesn't have a guide book full of blank pages. (LE has a lot of links in it that are blank).

The one issue I discovered with CE is you want to disable World Structure generation in it's config, because it shows up in the void world with new worlds.

I tore out LE and dropped in CE into an existing server, and it worked fine. Just had to re-buy my android abilities.
## Mod Details
* Name:
* CurseForge Link:
https://www.curseforge.com/minecraft/mc-mods/matter-overdrive-community-edition
